issue-40-built-in-benchmarking-tool.md

```diff
@@ -0,0 +1,31 @@
+## Add Built-in Benchmarking Tool
+
+## Summary
+
+There is a need to support a built-in benchmarking tool in Submariner.
```

Comment:
More context: this is to provide users with easy way to benchmark the network performance within and across clusters, and experiment with different tuning of the system.